config/default.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I would recommend to use typescript config file format.
https://github.com/lorenwest/node-config/wiki/Configuration-Files#typescript---ts

We can define a common interface, default values, which must be overloaded in each environment configuration.